perfkitbenchmarker/disk_iops_to_capacity.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IOPS Storage Utility class.
+
+This class is used to translate an {IOPS, Cloud Provider} requirement
+to {core, number of disks, storage size} machine requirements necessary
+to meet the IOPS level using the Cloud MySQL Provider declared.
+
+ - On AWS, we will use EBS performance levels.
```

Comment:
Update the comments:

on AWS, we will use the "ebs-gp2" storage type.

on GCP, we will use the PD-SSD storage type.

In future versions, we will support Azure, and more storage types from AWS and GCP.